terragrunt/aws/glue/etl/platform/gc_forms/process_data.py

```diff
@@ -122,6 +145,8 @@ def validate_with_gx(dataframe: pd.DataFrame, checkpoint_name: str) -> bool:
     gx_context_path = os.path.join(os.path.dirname(__file__), "gx")
     context = gx.get_context(context_root_dir=gx_context_path, cloud_mode=False)
 
+    configure_data_docs_site(context, TARGET_GX_BUCKET)
```

Comment:
I like this approach to dynamically decide if you want to save to the bucket or not